nixos/tests/all-tests.nix

```diff
@@ -288,8 +288,8 @@ in {
   activation-etc-overlay-mutable = runTest ./activation/etc-overlay-mutable.nix;
   activation-etc-overlay-immutable = runTest ./activation/etc-overlay-immutable.nix;
   activation-perlless = runTest ./activation/perlless.nix;
-  etcd = handleTestOn ["x86_64-linux"] ./etcd.nix {};
-  etcd-cluster = handleTestOn ["x86_64-linux"] ./etcd-cluster.nix {};
+  etcd = handleTestOn [ "aarch64-linux" "x86_64-linux" ] ./etcd/etcd.nix {};
```

Comment:
Do we still need `handleTestOn` any more or can this "just" become a `handleTest`?

Reply:
I lack the means of testing other platforms (particularly darwin). I can't say how it behaves there.
If you want, I can remove the platforms. I have no opinion on this. Just let me know the preference.

Reply:
OfBorg cannot test darwin, see:
https://github.com/NixOS/nixpkgs/pull/288801/checks?check_run_id=22982435118
So I'd rather keep this as it is.

Reply:
I think we should trust OfBorg for platforms.

Reply:
I have enabled tests for all plataforms. Then, I rely on OfBorg to test darwin.

Reply:
I have only tested "aarch64-linux". And I can only guarantee "aarch64-linux" works.